packages/@n8n/config/src/configs/database.ts

```diff
@@ -0,0 +1,159 @@
+import { Config, FromEnv, Nested } from '../decorators';
+
+@Config
+class LoggingConfig {
+	/** Typeorm logging enabled flag. @default false */
+	@FromEnv('DB_LOGGING_ENABLED')
+	readonly enabled: boolean = false;
+
+	/**
+	 * Logging level options, default is "error".
```

Comment:
Else we specify it 3 times:

```suggestion
	 * Logging level options.
```